python/ycm/vimsupport.py

```diff
@@ -301,6 +301,23 @@ def EscapedFilepath( filepath ):
   return filepath.replace( ' ' , r'\ ' )
 
 
+def TryJumpLocationInOpenedTab( filename, line, column ):
+  bufnr = GetBufferNumberForFilename( filename, False )
+  try:
+    tab, win = next( iter( [ ( tab, win ) for tab in vim.tabpages
+                                          for win in tab.windows
+                                          if tab != vim.current.tabpage
+                                          and win.buffer.number == bufnr ] ) )
+    vim.current.tabpage = tab
+    vim.current.window = win
+    vim.current.window.cursor = ( line, column - 1 )
```

Comment:
that line and the one below are copied from the original function `JumpToLocation` (I couldn't find a nice way to not duplicate those lines) but I didn't copied the comment with them. I'll add them.